templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/send_notification_email_for_released_statement.html.twig

```diff
@@ -3,11 +3,8 @@
 {%- set count = templateVars.statements|length -%}
 Guten Tag,
 
-im Verfahren "{{ templateVars.procedure.name|default('') }}"
-{{ 'email.text.notification_for_released_statement.1'|trans({ count: count }) }} freigegeben.
-
-Bitte prüfen Sie die {{ 'email.text.notification_for_released_statement.2'|trans({ count: count }) }}
-und reichen Sie sie bei dem Verfahrensträger ein.
+im Verfahren "{{ templateVars.procedure.name|default('') }}" {{ 'email.text.notification_for_released_statement.1'|trans({ count: count })|replace({ "\n": "" }) }} freigegeben.
```

Comment:
I don't get from where `\n` appears, that it has to be stripped away

Reply:
I think the line breaks come from the `translations.json` somehow? At least the `'\n'` match the line breaks in the `messages+intl-icu.de.yml`. See: `'email.text.notification_for_released_statement.1'`. But I don't know how to pack the contents of `'email.text.notification_for_released_statement.1' `in `the messages+intl-icu.de.yml` into one line? I have seen that other email texts are available completely as translation keys. Should this perhaps be adopted here too? See: `'fragment.notify.email.reviewer.admin.new'`.